Gemfile

```diff
@@ -46,6 +46,8 @@ group :development, :test do
   gem 'simplecov'
   gem 'vcr'
   gem 'webmock'
+  gem 'devise'
+  gem 'factory_bot'
```

Comment:
Thanks for setting this up!

Reply:
No problem I like learning more about factory bot so it was fun to re read the docs again.